8.x/core-concepts/cruds.md

```diff
@@ -118,204 +74,234 @@ class Todo extends BaseModel
 
-#### Step 2
----
-After adding the icon file, the address of the icon file should be added in the path `resources/assets/sass/icons.scss`.
+Put the following code in the path `/larammerce/resources/assets/sass/icons.scss`:
 
-##### EXAMPLE
-```php
-&.icon-todo {
-    background: url("/admin_dashboard/images/icons/todo.png");
-}
+```scss{8-10}
+.h-icon {
+    background-size: 100% 100% !important;
+    ...
+    &.icon-link-shortener {
+        background: url("/admin_dashboard/images/icons/link-shortener.png");
+    }
+
+    &.icon-todo {
+        background: url("/admin_dashboard/images/icons/todo-icon.png");
+    }
+    ...
 ```
 
-To build and minify the project resource files and save the given changes, the following command should be used in the terminal.
+Run the following command in the terminal:
 
-```bash
+```bash{1}
 npm run prod
 ```
 
-#### Step 3
----
-In the third stage, the **icon** and **icon path** must be introduced to the system in the path `config/cms/appliances.php`.
+Write the code below in the path `larammerce/config/cms/appliances.php`:
 
-##### EXAMPLE
-```php
-#todolist appliances
-[
-    "show_in_toolbar" => true,
+```php{8-17}
+<?php
+return [
+    ...
+    //analytic appliance
+    [
+        ...
+    ],
+    //todolist appliance
+    [
+        "show_in_toolbar" => true,
         "properties" => [
             "id" => "todo_management",
             "name" => "general.appliances.todo",
-            "icon" => "/admin_dashboard/images/icons/todo.png",
+            "icon" => "/admin_dashboard/images/icons/todo-icon.png",
             "route" => "admin.todo.index"
+        ],
     ],
-],
-``` 
+];
+```
 
-:::tip
-Pay attention that `show_in_toolbar` contains boolean data if the value is `true`, the icon will be displayed in the toolbar of the management panel,
-and if it is `false`, a new `appliance` must be defined in it.
-:::
+**OUTPUT**
+
+![todo_icon](/img-cruds/04-edited.png)
+
+
+#### Correct translation
+
+Insert the following code in the path `/larammerce/resources/lang/fa/general.php` inside the `appliances` section:
+
+```php{9}
+<?php
+return [
+    "appliances" => [
+        "setting" => "تنظیمات",
+        "shop" => "فروشگاه",
+        "directory" => "مدیریت فایل ها",
+        "analytic" => "تحلیل و بررسی",
+        "short_links" => "لینک های کوتاه",
+        "todo" => "لیست وظایف",
+    ],
+    ...
+];
+```
 
-Therefore, a todo folder should be created in the `larammerce/resources/views/admin/pages` path.
+Run the following command in the terminal:
+
+```bash{1}
+php artisan translation:fill
+```
+
+**OUTPUT**
+
+![todo_icon_translated](/img-cruds/08-edited.png)
 
-Blade files include the following:
 
-1. `index.blade.php`
+## Define index method
 
-2. `list.blade.php`
+If you click on todo icon to show the list of todos, you will get this error:
 
-3. `edit.blade.php`
+![index_error](/img-cruds/05.png)
```

Comment:
Not necessarily. In my system, when I tap the icon, the page just gets reloaded.  Please mention that it will not be correctly redirected and some errors may occur and remove this photo.